lib/improver/tests/utilities/test_cube_metadata.py

```diff
@@ -282,6 +284,15 @@ def test_coords_update_fails_bounds_differ(self):
         with self.assertRaisesRegex(ValueError, msg):
             update_coord(cube, self.coord_name, changes)
 
+    def test_update_attributes(self):
+        """Test update attributes associated with a coordinate."""
+        cube = create_cube_with_threshold()
+        changes = {'attributes': {'spp__relative_to_threshold': "above"}}
```

Comment:
Is the cube default not ```above```? If so this test is really checking that nothing has changed and should instead alter the attribute to ```below``` to show it works.

Reply:
I've updated this to be `below` as suggested.